docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+sharelatex:
+  image: sharelatex/sharelatex
+  links:
+    - mongo
+    - redis
+  ports:
+    - "80:80"
```

Comment:
Why should I?
The image itself exposes no specific ports and the application itself listens/works on port 80 - which is btw perfectly fine and expected for a web application. Exposing the port 5000 like in the other example is completely arbitrary and - IMHO - annoying for users accessing the service in their browser. (You might also want to use a reverse proxy or vhosts to make things easier for your users, but that requires a little more configuration.)

But feel free to change to the port to whatever you want to.

P.S. I have the feeling the project is not really interested in any kind of contribution from the community...
If the author is interested in exposing port 5000 I can modify the pull request, otherwise I will leave it as it is.